modules/react/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
-  "name": "@nebula.gl/overlays",
-  "description": "React overlays for deck.gl",
+  "name": "@deck.gl-community/react",
+  "description": "React components for deck.gl",
   "license": "MIT",
   "version": "1.0.4",
   "author": "Georgios Karnas <georgios@uber.com>",
```

Comment:
Just flagging so we don't forget, these will need to be updated at some point:

- author
- version (optional)
- main / module / exports / types

Reply:
OK, let's land and fix.